tests/NuGet.IndexingTests/TestSupport/MockSearcherManager.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Indexing;
+
+namespace NuGet.IndexingTests.TestSupport
+{
+    public class MockSearcherManager : NuGetSearcherManager
+    {
+        private string _indexName;
+
+        public MockSearcherManager(string indexName) 
+            : base(indexName: indexName, 
+                  logger: MockObjectFactory.CreateMockLogger().Object, 
+                  directory: null, 
+                  loader: null, 
+                  azureDirectorySynchronizer: null)
+        {
+            _indexName = indexName;
+            RegistrationBaseAddress[Constants.SchemeName] = new Uri(Constants.BaseUri);
```

Comment:
Removed this and changed to directly us it in the Searcher since it isn't used anywhere else.